projects/schematics/src/migrations/5_0/constructor-deprecations/data/generated-constructor.migration.ts

```diff
@@ -6131,9 +6327,13 @@ export const GENERATED_CONSTRUCTOR_MIGRATIONS: ConstructorDeprecation[] =
           importPath: '@spartacus/order/root',
         },
         {
-          className: 'ModalService',
+          className: 'LaunchDialogService',
```

Comment:
`ConsignmentTrackingComponent` is moved from `@spartacus/storefront` to `@spartacus/order/components`

Reply:
The move is properly registered in the doc and `rename-symbol.ts`